### DISCUSSION TITLE
Facebook comments

### DISCUSSION THREAD

**Discussion by Hariko50** [GetPublii/Publii#1894] [Help]
I have installed the plugin for Facebook comments. I don't understand how to enter comments on the website or how comments are displayed. I have activated that they are displayed.

**Comment by michaelfluegge** [comment]
You have both enabled comments and activated the plugin? https://getpublii.com/docs/facebook-comments.html

**Reply by Hariko50** [reply]
The website only contains the word Comments
and in Preview: FB Comments are not working in the preview mode
I think that indicates that it should actually work

Main Display comments enabled and this is the existing code:
`{{#if @config.post.displayComments}}
         <div class="content__comments">
            <div class="entry-wrapper">
               {{{@commentsCustomCode}}}
            </div>
         </div>
      {{/if}}
changed to:
{{#if @config.post.displayComments}}
    <div class="post__comments"> 
       <div class="wrapper"> 
          {{{@commentsCustomCode}}}
       </div> 
    </div> 
{{/if}}`

**Comment by Hariko50** [comment]
I have read that Facebook has restricted the use of this function in Germany, among other countries. I think that's my problem. I delete the plugin.
